### PR TITLE
chore: Correct the import location.

### PR DESCRIPTION
**Pull request by kinyoklion**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   "exports": {
     "types": "./dist/src/index.d.ts",
     "require": "./dist/launchdarkly-vue-client-sdk.umd.js",
-    "import": "./dist/launchdarkly-vue-client-sdk.es.js"
+    "import": "./dist/launchdarkly-vue-client-sdk.js"
```
Comment (kinyoklion) [inline package.json:20]: It seems that the newer version of Vite does not include the `.es`.

File: link-dev.sh [hunk 2]
```diff
@@ -21,6 +21,6 @@ do
   mkdir -p ${example}/node_modules/launchdarkly-vue-client-sdk/node_modules
   mkdir -p ${example}/node_modules/launchdarkly-vue-client-sdk/dist
   cp package.json ${example}/node_modules/launchdarkly-vue-client-sdk/package.json
-  cp -r node_modules/ ${example}/node_modules/launchdarkly-vue-client-sdk/node_modules/
```
Comment (kinyoklion) [inline link-dev.sh:24]: This used to exist in react-client-sdk as well. The original here happens to work on mac, but doesn't work on linux (you get dist/dist)

File: package.json [hunk 3]
```diff
@@ -14,6 +14,7 @@
     "node": ">=16.15.1"
   },
   "type": "module",
+  "module": "./dist/launchdarkly-vue-client-sdk.es.js",
```
Comment (kinyoklion) [inline package.json:17]: Added based on the Vite docs.

Comment (kinyoklion) [reply package.json:17]: The recommended package.json part: https://v2.vitejs.dev/guide/build.html#library-mode

File: vite.config.js [hunk 4]
```diff
@@ -7,7 +7,8 @@ export default defineConfig({
   build: {
     lib: {
       entry: 'src/index.ts',
-      name: 'launchdarkly-vue-client-sdk'
+      name: 'launchdarkly-vue-client-sdk',
+      fileName: (format) => `launchdarkly-vue-client-sdk.${format}.js`
```
Comment (kinyoklion) [inline vite.config.js:11]: New version doesn't do this by default, but there is an example in the docs.

**Review by louis-launchdarkly — APPROVED** [review]

**Pull request merged**